projects/zookeeper.cmake

```diff
@@ -142,7 +142,7 @@ function(build_zookeeper)
     xpNewDownload(${CPP_UNIT})
 
     set(ACLOCAL_STR "aclocal -I ${SOURCE_DIR}/src/c/cppunit-${CPP_UNIT_VER}")
-    ExternalProject_Add(zookeeper_configure DEPENDS zookeeper download_cppunit-${CPP_UNIT_VER}.tar.gz
+    ExternalProject_Add(zookeeper_configure DEPENDS zookeeper_ant download_cppunit-${CPP_UNIT_VER}.tar.gz
```

Comment:
Hmm, I swear I had tried that before and ran into some issues, but I'm most likely remembering wrong.